pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
> I don't think NERRS is being sloppy per se 

I did not mean that.  I am coming  from a code-review point of view where we have a non-documented assumption there (it is positive and must be negative).

Although one might argue that I partially fixed it because we still did not document that they must be negative :stuck_out_tongue_winking_eye: